config/default-amo.js

```diff
@@ -24,6 +24,21 @@ module.exports = {
   },
   enableTrailingSlashesMiddleware: true,
   fxaConfig: 'amo',
+
+  // This needs to be kept in sync with addons-server's SUPPORTED_NONLOCALES
+  // settings value: https://github.com/mozilla/addons-server/blob/master/src/olympia/lib/settings_base.py#L274
+  validRootUrlExceptions: [
```

Comment:
> Would it be clearer to keep the config keys between the server and frontend more similar? Having the locale in the name makes it a bit clear what this does to me somehow.

Yeah, I actually misunderstood this a bit because during testing I saw some URL redirects and didn't realise one was to the CDN. I need to change the patch a bit.